index.html

```diff
@@ -66,31 +66,159 @@ <h3 class="pipeline-header__title">In the pipeline</h3>
       <br> in DeFi</h2>
     <div class="fundraising-list">
       <div class="fundraising-item">
-        <div class="list-item__number">01</div>
-        <h3 class="list-item__text">Without a big budget, it's hard to <br> approach venture investors</h3>
+        <div class="fundraising-item__number">01</div>
+        <p class="fundraising-item__text">Without a big budget, it's hard to <br> approach venture investors</p>
       </div>
       <div class="fundraising-item">
-        <div class="list-item__number">02</div>
-        <h3 class="list-item__text">VCs don't care about your DeFi idea <br> – only their gains</h3>
+        <div class="fundraising-item__number">02</div>
+        <p class="fundraising-item__text">VCs don't care about your DeFi idea <br> – only their gains</p>
       </div>
       <div class="fundraising-item">
-        <div class="list-item__number">03</div>
-        <h3 class="list-item__text">They force you into selling your <br> tokens at a huge discount…</h3>
+        <div class="fundraising-item__number">03</div>
+        <p class="fundraising-item__text">They force you into selling your <br> tokens at a huge discount…</p>
       </div>
       <div class="fundraising-item">
-        <div class="list-item__number">04</div>
-        <h3 class="list-item__text">...Then dump them at the first <br> opportunity</h3>
+        <div class="fundraising-item__number">04</div>
+        <p class="fundraising-item__text">...Then dump them at the first <br> opportunity</p>
       </div>
       <div class="fundraising-item">
-        <div class="list-item__number">05</div>
-        <h3 class="list-item__text">Result: your token collapses. <br> Everyone goes home.</h3>
+        <div class="fundraising-item__number">05</div>
+        <p class="fundraising-item__text">Result: your token collapses. <br> Everyone goes home.</p>
       </div>
     </div>
   </div>
 </section>
+<section class="vision">
+  <div class="max-width vision__inner">
+    <div class="vision-header">
+      <h3 class="subtitle">— For Projects</h3>
+      <h2 class="vision-title h2">Vision is the <span class="highlight">answer</span></h2>
+      <p class="vision-text">You have an idea, IncuPie has a plan. We offer everything you need <br> to go from an
+        early-stage idea to a successful exchange listing.</p>
+    </div>
+    <div class="benefit-list">
+      <div class="benefit-row">
+        <div class="benefit-item">
+          <img alt="checkmark" src="images/checkmark.svg">
+          <div class="vision-item__text">
+            <h3 class="benefit-title">Decentralized investments</h3>
+            <p class="vision-item__subtitle">we'll distribute your tokens among our huge and active <br> community</p>
+          </div>
+        </div>
+        <div class="benefit-item">
+          <img alt="checkmark" src="images/checkmark.svg">
+          <div class="vision-item__text">
```

Comment:
if you still need to have a specific class here, I think it would be better to rename it to `vision-item__content`